.github/workflows/release.yml

```
@@ -231,18 +234,22 @@ jobs:
 
       - name: Build release artifacts (manual)
         if: ${{ !startsWith(github.ref, 'refs/tags/') }}
-        run: pnpm tauri build --target ${{ matrix.target }} ${{ matrix.tauri_args }}
+        run: pnpm tauri build --target "$MATRIX_TARGET" $TAURI_ARGS
         env:
+          MATRIX_TARGET: ${{ matrix.target }}
+          TAURI_ARGS: ${{ matrix.tauri_args }}
           CARGO_TARGET_ARMV7_UNKNOWN_LINUX_GNUEABIHF_LINKER: ${{ matrix.linker }}
```

actionlint (1.7.12)

[error] property "linker" is not defined in object type {os: string; platform: string; target: string; tauri_args: string}

(expression)